### DISCUSSION TITLE
Horizontal line

### DISCUSSION THREAD

**Discussion by matteocorti** [RomRider/apexcharts-card#310] [Q&A] [answered]
Hi,
is there a way to draw a horizontal line (to show a threshold)?
E.g. a line to show a given temperature threshold.

**Comment by upwindanderl** [comment, marked as answer]
Like this: https://github.com/RomRider/apexcharts-card/issues/328

https://community.home-assistant.io/t/apexcharts-card-a-highly-customizable-graph-card/272877/653?u=romrider
